developers/academy/zero_to_mvp/101_hello_weaviate/20_examples_1.mdx

```diff
@@ -181,7 +190,7 @@ Take a look at this response. Do you notice any additional information?
 
 </details>
 
-The difference is that the response now contains a `distance` value.
+The difference is that these responses contain a `distance` value.
```

Comment:
I think this should remain singular as there is just the one (represented in different ways). But I am being a real pedant. Happy to go with whatever.